### ISSUE TITLE
Ability to retrieve client actions/packets

### ISSUE THREAD

**Issue by TheMode** [ReplayMod/ReplayStudio#16]
I am currently looking at ReplayStudio with the goal of using replays in junit tests for an anti-cheat, the fact that server-bound packets are not recorded make it a bit confusing.

Are self-movement packets automatically converted to their client-bound equivalent? What about including client packets in the API?

**Comment by Johni0702** [comment]
Yes, the mod synthesizes client-bound packets for any actions which it know the server won't echo back.
Once the replay is recorded, the recording player entities is (conceptually, technically it probably gets updates at different rates, etc.) no different than any other player.

Server-bound packets are never even recorded because they cannot be played back (converting them to client-bound ones on the fly is significantly more difficult than synthesizing server-bound ones during recording), so there's been no use for them.

**Comment by TheMode** [comment]
Are those packets listed somewhere? The wiki page mentioned in the readme is not accessible (https://github.com/ReplayMod/ReplayStudio/wiki)
